pkg/users/local.js

```diff
@@ -318,6 +324,8 @@ function log_unexpected_error(error) {
 PageAccounts.prototype = {
     _init: function() {
         this.id = "accounts";
+        this.permission = cockpit.permission({ admin: true });
```

Comment:
I think we need to use the pkg/lib/superuser.jsx since last Marius changes.

Reply:
I don't think that superuser actually has user. Does it? If I saw correctly, there is only `allowed`.